jira2markdown/parser.py

```diff
@@ -17,4 +17,5 @@ def convert(text: str, usernames: Optional[dict] = None, elements: Optional[Mark
     inline_markup << elements.expr(inline_markup, markup, usernames, filter(lambda e: e.is_inline_element, elements))
     markup << elements.expr(inline_markup, markup, usernames, elements)
 
+    text = text.replace("\r\n", "\n")
```

Comment:
@catcombo: This currently works for me in my production workflow, however I would consider accounting for individual Carriage Returns (CR) )`\r`). It is not an edge case that currently breaks something in my workflow but could for someone else since it was a valid new-line on [legacy Mac's](https://stackoverflow.com/questions/1761051/difference-between-n-and-r#:~:text=in%20old%20Mac%20systems%20(pre%2DOS%20X)%2C%20%5Cr%20was%20the%20code%20for%20end%2Dof%2Dline%20instead).

```python
    text = text.replace("\r\n", "\n").replace("\r", "\n")
```

Reply:
This is what I afraid of. I so not sure about these replacements. Probably this week I'll have more time to investigate how to do it with `pyparsing`. Lets wait until the end of the week.

Reply:
Alright, I found how to keep existing line breaks and reuse them if possible. Now in most of the cases line breaks in the converted string will be consistent. @arctus-io can you test it on your side?